src/tfr_dumping/src/dumping_action_server.cpp

```diff
@@ -94,13 +93,13 @@ class Dumper
         {  
             ROS_INFO("dumping action server started dumping procedure");
             //check to make sure we can see the board
-            tfr_msgs::ArucoResult initial_estimate{};
-            getArucoEstimate(initial_estimate);
-            if (initial_estimate.number_found == 0)
-            {
-                server.setAborted();
-                return;
-            }
+//            tfr_msgs::ArucoResult initial_estimate{};
+//            getArucoEstimate(initial_estimate);
+//            if (initial_estimate.number_found == 0)
+//            {
+//                server.setAborted();
+//                return;
+//            }
```

Comment:
Were these commented out for testing? If so, that change probably shouldn't be merged into the codebase.